Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM golang:1.18.3-bullseye as build
+FROM golang:1.18.3-alpine as build
 
 RUN mkdir /app
 WORKDIR /app
 
+RUN apk update && \
+    apk add --no-cache build-base git make
```

Comment:
Now that we list `git` explicitly, we should remove `build-base`, shouldn't we?

Reply:
Unfortunately I wasn't able to fix the Go build without using `build-base` :/

Reply:
Nevermind, you tried it 😊